authentication/src/main/java/life/qbic/authentication/application/notification/Notification.java

```diff
@@ -1,7 +1,7 @@
-package life.qbic.shared.application.notification;
+package life.qbic.authentication.application.notification;
```

Comment:
Isnt the notification and the notification service something broader and should be part of the messaging of the software (newsreader + broadcasting)?